### PR TITLE
Fusion Rebalance: April Fools edition

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
As we all know fusion is inherently weak and deserves a buff so I have concocted this pr to remedy this. 

For obvious reasons, this should only be merged on April fools for maybe a few rounds to see the funny rads. For those attempting to contain a fusion of this magnitude, I would recommend checking what the fusion particles now pass through and don't pass through to find some form of a solution. 

## Why It's Good For The Game
the sun (2)

## Testing Photographs and Procedure

<details>
I found this dog (I'm assuming I don't need to show testing procedure and photographs since this is a meme pr intended for like a few rounds on April fools) 

Mammoth

</details>

## Changelog
:cl: Phil Smith, Allah
tweak: Try fusion at your own risk (APRIL FOOLS!!)
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
yes

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	set_light(4, 3, particle_colors[our_color]) //Range of 4, brightness of 3 - Same range as a flashlight
 
 /obj/item/projectile/energy/nuclear_particle/proc/customize(custompower)
-	irradiate = max(3000 * 3 ** (log(10,custompower)-FUSION_RAD_MIDPOINT),10)
+	irradiate = INFINITY // trolle!!
```
Comment (MNarath1) [inline code/modules/projectiles/projectile/energy/nuclear_particle.dm:28]:
```suggestion
	irradiate = custompower // trolle!!
```

Comment (MNarath1) [inline code/modules/projectiles/projectile/energy/nuclear_particle.dm:28]: Actually even with this that high nuclear particle won't deal more than 30 damage look at the switch below still applies crazy rads trough

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm [hunk 2]
```diff
@@ -2,10 +2,10 @@
 /obj/item/projectile/energy/nuclear_particle
 	name = "nuclear particle"
 	icon_state = "nuclear_particle"
-	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE
+	pass_flags = PASSTABLE | PASSGLASS | PASSGRILLE | PASSDOORS | PASSCLOSEDTURF | PASSFLAPS | PASSMACHINE
 	damage = 10
 	damage_type = TOX
-	irradiate = 2500 //enough to knockdown and induce vomiting
+	irradiate = INFINITY //:)
```
Comment (MNarath1) [inline code/modules/projectiles/projectile/energy/nuclear_particle.dm:8]:
```suggestion
	irradiate = 2500 //enough to knockdown and induce vomiting
```

Comment (MNarath1) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm:8]: You already modify the radiation to its crazy level in the proc below.

**Review by MNarath1 — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm [hunk 3]
```diff
@@ -52,7 +52,7 @@
 			name = "impossibly strong nuclear particle"
 			damage = 30
 
-/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = 1e12) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
+/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = INFINITY) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
```
Comment (MNarath1) [inline code/modules/projectiles/projectile/energy/nuclear_particle.dm:55]:
```suggestion
/atom/proc/fire_nuclear_particle(angle = rand(0,360), customize = FALSE, custompower = 1e12) //used by fusion to fire random nuclear particles. Fires one particle in a random direction.
```

Comment (MNarath1) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm:55]: Just don't bother this is probably beeing overwriten by the proccal anyways so lets just apply infinity down the procchain instead

**Review by MNarath1 — CHANGES_REQUESTED** [review]

**Comment by TeomanTheGreat** [comment]
Since no maintainer man has looked at this and its 2 am in my timezone (no longer april fools) I shall close this.

**Pull request closed without merging**